codespell_lib/data/dictionary.txt

```diff
@@ -1825,6 +1827,10 @@ analoguously->analogously
 analogus->analogous
 analouge->analogue
 analouges->analogues
+analsye->analyse
+analsyed->analysed
+analsyer->analyser
+analsying->analysing
```

Comment:
@luzpaz if you do not mind adding these
```text
analsyers->analysers
analsyes->analyses
analsyt->analyst
analsyts->analysts
```
* https://github.com/search?q=analsyers&type=code
* https://github.com/search?q=analsyes&type=code
* https://github.com/search?q=analsyt&type=code
* https://github.com/search?q=analsyts&type=code